website/content/en/status/report-2023-04-2023-06/gcc.adoc

```diff
@@ -19,12 +19,14 @@ With the release of GCC 13.1 (first GCC 13 release: I remind that GCC counts min
 
 ==== The *-devel ports
 
-A few significant changes have been made on the *-devel ports, which will be forwarded to the production ports together with more changes (see below):
+Enabled support for .init_array and .fini_array, which FreeBSD supports since commit gitref:83aa9cc00c2d83d05a0efe7a1496d8aab4a153bb[repository=src].
 
-* enable support for .init_array and .fini_array, which FreeBSD supports since commit gitref:83aa9cc00c2d83d05a0efe7a1496d8aab4a153bb[repository=src];
+Reverted from LTO_BOOTSTRAP to STANDARD_BOOTSTRAP as the default option on i386, amd64 and aarch64: 
```

Comment:
I would change it into:

"The default bootstrap option on i386, amd64, and aarch64 has been reverted from LTO_BOOTSTRAP to STANDARD_BOOTSTRAP:"